feat(spring-boot-starter): add setting tenantId in @Deployment annotation (#7238)
camunda/camunda-docs: docs/apis-tools/camunda-spring-boot-starter/configuration.md

@Deployment(resources = "classpath*:/bpmn/**/*.bpmn")
```

To adjust the tenant to deploy to, set the `tenantId` property of the `@Deployment` annotation:

```java
@Deployment(resources = "classpath:demoProcess.bpmn", tenantId = "myTenant")
public class MyRandomBean {
  // make sure this bean is registered
}
```

> By default, the tenant id being set to `camunda.client.tenant-id` will be used.

Reviewer: ```suggestion
By default, the `tenantId` set to `camunda.client.tenant-id` is used.
```
